test/release/examples/benchmarks/shootout/regexdna-redux.good

```diff
@@ -1,3 +1,5 @@
+regexdna-redux.chpl:13: In function 'main':
+regexdna-redux.chpl:32: warning: 'readstring' is deprecated; please use 'readString' instead
```

Comment:
@jeremiah-corrado : Didn't catch this in time for 1.30, but copies of the shootout benchmarks in the release (or really, any directories other than `test/studies/shootout/submitted/*` should be updated w.r.t. deprecations like this.  The reason the `.../submitted/` versions are special is so that (a) we can continue to track the performance of the versions that are live on the website and (b) we can update those versions when needed.  Note regexdna.good in this same directory can be similarly updated.

Reply:
Okay, I must have forgot about that when I made this change, or I was being overly cautious. I'll put in a fix after the release.

Reply:
This is fixed now: https://github.com/chapel-lang/chapel/pull/22045

Reply:
Thanks!